ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -96,80 +96,80 @@ public final class AliasTokens: NSObject {
     public var shadow: TokenSet<ShadowTokens, ShadowInfo> = .init { token in
         switch token {
         case .clear:
-            return ShadowInfo(colorOne: DynamicColor(light: ColorValue.clear),
-                              blurOne: 0.0,
-                              xOne: 0.0,
-                              yOne: 0.0,
-                              colorTwo: DynamicColor(light: ColorValue.clear),
-                              blurTwo: 0.0,
-                              xTwo: 0.0,
-                              yTwo: 0.0)
+            return ShadowInfo(ambientColor: DynamicColor(light: ColorValue.clear),
+                              ambientBlur: 0.0,
+                              xAmbient: 0.0,
+                              yAmbient: 0.0,
+                              keyColor: DynamicColor(light: ColorValue.clear),
+                              keyBlur: 0.0,
+                              xKey: 0.0,
+                              yKey: 0.0)
         case .shadow02:
-            return ShadowInfo(colorOne: DynamicColor(light: ColorValue(r: 0.0, g: 0.0, b: 0.0, a: 0.14),
+            return ShadowInfo(ambientColor: DynamicColor(light: ColorValue(r: 0.0, g: 0.0, b: 0.0, a: 0.14),
```

Comment:
So looking at the actual values, `key` and `ambient` should be switched. The ambient shadow does not have any x or y offset - only the key shadows will have an offset.

You can refer to https://github.com/microsoft/fluentui-react-native/blob/main/packages/experimental/Shadow/SPEC.md for what all the values are in FURN.